docs/website/docs/guides/deployment-configurations/gpu-rocm.md

```diff
@@ -45,14 +45,14 @@ the IREE compiler, then enable the ROCm compiler target with the
 
 ### Get the IREE runtime
 
-Next you will need to get an IREE runtime that includes the ROCm HAL driver.
+Next you will need to get an IREE runtime that includes the HIP HAL driver.
```

Comment:
Thanks for the doc updates. I was also looking at these. Found a few other places to touch up: https://github.com/ScottTodd/iree/commit/c728bec69d2b820f99cec6e43502048fe1172c00

(I'm sort of waiting for a ROCM-capable dev machine so I can personally go through the setup instructions myself... it's hard to update docs with confidence when only working from remote machines that are already configured)

Reply:
I just changed these main build commands so we're up to date if any user wants to build targeting AMD. If you're working on a comprehensive setup instructions overhaul, I can hold off on these changes

Reply:
I'm not currently working on it. I started that commit then paused when I determined that it would be way easier with a local dev environment... which I might have in a few weeks. Help refreshing the docs would be great if you're up for it. Otherwise it'll be in my backlog.

Reply:
Oh sure, I can take it up. Will do a follow up patch with (if any) updated instructions

Reply:
This file also has some references to the `rocm` HAL driver/device: https://github.com/iree-org/iree/blob/5aaaa79c30601985d12a0cdd081a6a3884ce55d8/docs/website/docs/guides/deployment-configurations/index.md?plain=1#L99